tests/unit/src/test/scala/tests/DebugProtocolSuite.scala

```diff
@@ -0,0 +1,104 @@
+package tests
+import java.util.Collections.emptyList
+import java.util.concurrent.TimeUnit.SECONDS
+
+import ch.epfl.scala.bsp4j.DebugSessionParamsDataKind
+import ch.epfl.scala.bsp4j.ScalaMainClass
+
+import scala.meta.internal.metals.MetalsEnrichments._
+
+object DebugProtocolSuite extends BaseSlowSuite("debug-protocol") {
+  private val mainFile = "a/src/main/scala/a/Main.scala"
+
+  testAsync("start") {
+    for {
+      _ <- server.initialize(
+        s"""/metals.json
+           |{
+           |  "a": {}
+           |}
+           |/$mainFile
```

Comment:
Can we replace `mainFile` withe actual file, it would be a bit clearer and all code in one place.